rust-src/concordium_base/src/id/sigma_protocols/com_ineq.rs

```diff
@@ -1,17 +1,17 @@
 //! The module provides the implementation of the sigma protocol for "proof of
-//! inequality for committed value and public value". This protocol enables one
-//! to prove that a committed value is not equal to a public one, without
-//! revealing the value.
+//! inequality for committed value and public value" (cf. Section 9.2.13,
+//! Bluepaper v1.2.5). This protocol enables one to prove that a committed value
```

Comment:
I think this sort of referencing is very limited in value. Soon you'll change the section numbering because a new section is added or some such, and this reference will be very hard to find (in a year, will you really bother looking up an exact version of the bluepaper?). It should be possible to find the protocol from its name, I'd suggest.

Reply:
I have updated the references to include the section title which rarely changes.